CompanionApp/Companion.iOS/Services/NativeAudio.cs

```diff
@@ -0,0 +1,47 @@
+﻿using AVFoundation;
+using Companion.iOS.Services;
+using Companion.Services;
+using Foundation;
+using System;
+using Xamarin.Forms;
+
+[assembly: Dependency(typeof(NativeAudio))]
+namespace Companion.iOS.Services
+{
+    public class NativeAudio : IRecordingInterface
+    {
+        public NativeAudio()
+        {
+        }
+
+        public void RouteAudioToSpeaker()
+        {
+            var audioSession = AVAudioSession.SharedInstance();
+            var success = audioSession.SetCategory(AVAudioSession.CategoryPlayAndRecord,
+                                                   out NSError error);
+            if (success)
+            {
+                Console.WriteLine("Successfully set audio category to 'PlayAndRecord'");
+                success = audioSession.OverrideOutputAudioPort(AVAudioSessionPortOverride.Speaker, out error);
+                if (success)
+                {
+                    Console.WriteLine("Successfully overrode audio to speaker");
+                    success = audioSession.SetActive(true, out error);
+
+                    if (success)
+                    {
+                        Console.WriteLine("Successfully set new audio state to Active");
+                    }
+                }
+            }
```

Comment:
What happens with all of these if it doesn't succeed at any of these steps? Does there need to be an error message?